src/App.js

```diff
@@ -85,17 +48,8 @@ export default function App() {
           path="/profile"
           element={
             <ProfilePage
-              creepypastas={creepypastas}
-              nameEditingValue={nameEditingValue}
-              infoEditingValue={infoEditingValue}
-              onNameChange={onNameChange}
-              onInfoChange={onInfoChange}
-              onNameBlur={onNameBlur}
-              onInfoBlur={onInfoBlur}
-              onKeyDown={onKeyDown}
-              profileImage={profileImage}
-              profileUpload={profileUpload}
               handleBookmarkClick={handleBookmarkClick}
```

Comment:
Since you are passing the function, it would be correct to use the prefix "on" after convention.